backend/src/main/java/com/htecgroup/skynest/controller/UserController.java

```diff
@@ -284,10 +277,8 @@ public ResponseEntity<String> confirmPasswordReset(
   @PreAuthorize("hasAuthority(T(com.htecgroup.skynest.model.entity.RoleEntity).ROLE_WORKER)")
```

Comment:
I know this is not part of your PR, but why is this method annotated with Role-Worker ? isn't it already by default ?

Reply:
It was, rebased too early by accident. Will check with the team if that is necessary